gradoop-common/src/main/java/org/gradoop/common/model/impl/properties/PropertyValue.java

```diff
@@ -148,14 +137,9 @@
   private static final long serialVersionUID = 1L;
```

Comment:
It is advised to manually define a serialVersionUID because otherwise, it will be generated by the compiler. The generated UID depends on the structure of the class, so if the structure would change, serialized instances could not be deserialized. Altough this might not be an issue, i still think it is better to just define a UID.